### PR TITLE
Fix: Indent rule error on empty block body (fixes #2999)

### LINKED ISSUE #2999: "indent" rule throws "TypeError: Cannot read property 'start' of undefined"
Using the code on `master` (as of https://github.com/eslint/eslint/commit/1c70834295b064021e6aa6698473375063bffbae), I get this crash with the `indent` rule enabled:

```
TypeError: Cannot read property 'start' of undefined
    at getNodeIndent (/Users/feross/pr/standard/node_modules/eslint/lib/rules/indent.js:72:51)
    at /Users/feross/pr/standard/node_modules/eslint/lib/rules/indent.js:102:30
    at Array.forEach (native)
    at checkNodesIndent (/Users/feross/pr/standard/node_modules/eslint/lib/rules/indent.js:101:15)
    at EventEmitter.blockIndentationCheck (/Users/feross/pr/standard/node_modules/eslint/lib/rules/indent.js:254:9)
    at EventEmitter.emit (events.js:129:20)
    at Controller.controller.traverse.enter (/Users/feross/pr/standard/node_modules/eslint/lib/eslint.js:783:25)
    at Controller.__execute (/Users/feross/pr/standard/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/feross/pr/standard/node_modules/eslint/node_modules/estraverse/estraverse.js:495:28)
    at EventEmitter.module.exports.api.verify (/Users/feross/pr/standard/node_modules/eslint/lib/eslint.js:776:24)
```

You can trigger the error with the code:

```js
if (true) {
  // This is a comment
}
```

and the rule:

```
"indent": [2, 2]
```

### PR DESCRIPTION
**Pull request by gyandeeps**

### CODE REVIEW

File: tests/lib/rules/indent.js [hunk 1]
```diff
@@ -40,6 +40,25 @@ eslintTester.addRuleTest("lib/rules/indent", {
     valid: [
         {
             code:
+            "if (1 < 2){\n" +
+            "//hi sd \n" +
```
Comment (ilyavolodin) [inline tests/lib/rules/indent.js]: Should the indentation of comments be checked as well?

Comment (gyandeeps) [inline tests/lib/rules/indent.js]: I am not sure as this was never part of indent concept from the beginning as far as I understand.

Comment (ilyavolodin) [inline tests/lib/rules/indent.js]: Sounds good. Let's merge as is, and we can think about comments later.

Comment (gyandeeps) [inline tests/lib/rules/indent.js]: sounds good

**Pull request merged**